### PR TITLE
Add script to build BOOT.BIN for versal

### PR DESCRIPTION
**Pull request by AAndrisa**

### CODE REVIEW

**Review by SRaus — CHANGES_REQUESTED** [review]

**Comment by AAndrisa** [comment]
v2: PR changes added

**Comment by AAndrisa** [comment]
v3: fix indentation

File: versal_boot_bin/build_versal_boot_bin.sh [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+	build_u_boot
+)
+	cp $UBOOT_FILE/u-boot.elf $OUTPUT_DIR/u-boot.elf
+elif [ "$UBOOT_FILE" == "download" ]; then
+(
+	cd $BUILD_DIR
+	git clone https://github.com/Xilinx/u-boot-xlnx
+	cd u-boot-xlnx
+	build_u_boot
+	cp u-boot.elf ../../$OUTPUT_DIR/u-boot.elf
```
Comment (SRaus) [inline versal_boot_bin/build_versal_boot_bin.sh:68]: Just make all of the copy commands consistent - see #61, #68, #76 etc (all inside or all outside parenthesis)
I think that on line #68 is  needed one single level up (../) not two (../../). Please check.

**Comment by AAndrisa** [comment]
v4: update cp path

**Review by SRaus — APPROVED** [review]

**Pull request merged**